### PR TITLE
PLATML-5950 Incorporate sandbox-related logic into Python sample

### PR DESCRIPTION
**Pull request by mpaunovi**

### CODE REVIEW

**Comment by tumulurik** [comment]
Milos, can you please base your changes on this branch - https://github.com/tumulurik/acp-data-services-dsw-reference/tree/PLATML-5317/recipes/python/retail/retail 
This has the changes for the platform SDK based on platform SDK 0.0.9 
This branch has been blocked for merging because of a bug on INT and Stage envs and thats is why it is sitting in the branch.

**Comment by tumulurik** [comment]
Wait on approving this PR - cannot be merged until the SDK and hence the runtime are updated.

File: recipes/python/retail/Dockerfile
Comment (mpaunovi) [inline recipes/python/retail/Dockerfile]: Until new runtime version with latest python SDK is released.
https://jira.corp.adobe.com/browse/PLATML-5970

**Comment by tumulurik** [comment]
The changes LGTM, ready to approve once the SDK version is updated and testing with this is complete.

**Review by NemanjaL94 — APPROVED** [review]

**Review by tumulurik — APPROVED** [review]

**Pull request merged**
